extensions/Worker.Extensions.CosmosDB/src/WorkerCosmosSerializer.cs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+// Adapted from: https://github.com/Azure/azure-cosmos-dotnet-v3/blob/master/Microsoft.Azure.Cosmos/src/Serializer/CosmosJsonDotNetSerializer.cs
+
+using System;
+using System.IO;
+using System.Text;
+using Azure.Core.Serialization;
+using Microsoft.Azure.Cosmos;
+
+namespace Microsoft.Azure.Functions.Worker
+{
+    /// <summary>
+    /// This is a wrapper that allows us to use the worker options ObjectSerializer to create a CosmosSerializer.
+    /// <summary>
+    public sealed class WorkerCosmosSerializer : CosmosSerializer
```

Comment:
Unless we have a good reason for `public`, I recommend `internal`.

```suggestion
    internal sealed class WorkerCosmosSerializer : CosmosSerializer
```